std/experimental/allocator/gc_allocator.d

```diff
@@ -110,3 +129,24 @@ unittest
     version (Windows) { /* FIXME:, TODO: test fails on win32 auto-tester */ }
     else assert(GCAllocator.instance.expand(b, 1));
 }
+
+unittest
+{
+    import core.memory: GC;
+
+    // test allocation sizes
+    assert(GCAllocator.instance.goodAllocSize(1) == 16);
+    for(size_t s = 16; s <= 8192; s *= 2)
+    {
+        assert(GCAllocator.instance.goodAllocSize(s) == s);
+        assert(GCAllocator.instance.goodAllocSize(s - (s / 2) + 1) == s);
+
+        auto buffer = GCAllocator.instance.allocate(s);
```

Comment:
Wouldn't be better to request `s - (s / 2) + 1` here instead of `s`? That way you do make sure that the allocator gets you the extra bytes up to `s`.